c2nim.nim

```diff
@@ -43,6 +43,10 @@ Options:
   --importc              annotate procs with ``{.importc.}``
   --importdefines        import C defines as procs or vars with ``{.importc.}``
   --importfuncdefines    import C define funcs as procs with ``{.importc.}``
+  --def:SYM='macro()'    define a C macro that gets replaced with the given
+                         definition. It's parsed by the lexer. Use it to fix
+                         function attributes: ``--def:PUBLIC='__attribute__ ()'``
+  --reordercomments      reorder C comments to match Nim's postfix style
```

Comment:
does this really need to be an option? ie why not always do this (since it's needed to make the comments compatible with nim doc)

Reply:
Two reasons: 1) to make it easier to get accepted 2) because I’d have to reorder the comments in all the tests. 

Though perhaps it could default to on and use the option to disable it. Plus there may be cases it’s wrong. I sorta hack the new lines with a heuristic.